<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>student_score</class>
 <widget class="QWidget" name="student_score">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,7,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,8">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_expandclass">
         <property name="text">
          <string>点击展开班级</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_class"/>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_score"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,4,2,0">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_average">
         <property name="text">
          <string>当前班级平均分：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_passrate">
         <property name="text">
          <string>当前班级及格率：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_okinscore">
         <property name="text">
          <string>确定导入成绩</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>导出成绩表为CSV文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
